### PR TITLE
Updates.

### PR DESCRIPTION
**Pull request by mrrpnya**
Added:
- Bootstrap 5.3 CSS Framework
- Custom SVG-format icons.
- Syntax highlighting for future
- Added Contributors
- Added Contributor Page
- Added Social
- Enabled Jekyll
- Added Windows/Linux shell files (Batch, Shellscript) for quick launching of the server.
- Added Navigation bar

Changed:
- Moved Catppuccin theming data to `catppuccin_mocha.css`
- Restructured `main.css`
- Restructured header
- Changed header icon (large Trans Pride icon to reasonably sized Pride icon)

Removed:
- Footer

To do:
- Make navigation mobile friendly (Hamburger context menu)
- Provide details about the project, its structure, design requirements, goals, etc.
- etc

### CODE REVIEW

**Comment by sparkles-laurel** [comment]
can i merge this

**Comment by sparkles-laurel** [comment]
can i merge this

**Comment by electron271** [comment]
please wait for a review

**Comment by electron271** [comment]
(or make a review)

File: README.md [hunk 1]
```diff
@@ -65,20 +65,20 @@ To run the kernel in QEMU, run the following command:
...
-  
-  | [SparkySimp](https://github.com/SparkySimp) | [electron271](https://github.com/electron271) | [SatoTsukasaCode](https://github.com/SatoTsukasaCode) | [Techiesplash](https://github.com/Techiesplash) | [Lily8000](https://github.com/Lily8000) |
-  
-  | Lead Developer | Lead Developer | Developer | Developer | Documentation |
-  
+| <img src="https://github.com/SparkySimp.png" width="100" height="100"> | <img src="https://github.com/electron271.png" width="100" height="100"> | <img src="https://github.com/SatoTsukasaCode.png" width="100" height="100"> | <img src="https://github.com/Techiesplash.png" width="100" height="100"> | <img src="https://github.com/Lily8000.png" width="100" height="100"> |
+
+|:---:|:---:|:---:|:---:|:---:|
+
+| [SparkySimp](https://github.com/SparkySimp) | [electron271](https://github.com/electron271) | [SatoTsukasaCode](https://github.com/SatoTsukasaCode) | [Techiesplash](https://github.com/Techiesplash) | [Lily8000](https://github.com/Lily8000) |
```
Comment (sparkles-laurel) [inline README.md:72]: We have forgotten to give credit to @HackedOS here who is a significiant contributor to our kernel

**Review by sparkles-laurel — APPROVED** [review]

**Comment by sparkles-laurel** [comment]
i freaking
approved it by accident

**Comment by mrrpnya** [comment]
ROFL.

**Comment by mrrpnya** [comment]
Also can't add the credit right now, very far away from the keyboard.

I'd fix that with another commit+PR if needed.

**Comment by sparkles-laurel** [comment]
i can actually handle that myself

**Pull request merged**